tests/e2e/assets/webpack/test-server-app/app/injectable.ts

```diff
@@ -4,5 +4,5 @@ import {DOCUMENT} from '@angular/platform-browser';
 
 @Injectable()
 export class MyInjectable {
-  constructor(public viewContainer: ViewContainerRef, @Inject(DOCUMENT) public doc) {}
```

Comment:
This test is done in the normal webpack tests.

This test makes my tests fail because I am actually executing the application and it fails to bootstrap because the service is not able to find the VCR.

Is it fine to remove it from this set of tests because the functionality is covered in the other webpack tests?